posixlib/src/main/scala/scala/scalanative/posix/time.scala

```diff
@@ -12,37 +12,64 @@ object time {
   type timespec = CStruct2[time_t, CLong]
   type tm       = CStruct9[CInt, CInt, CInt, CInt, CInt, CInt, CInt, CInt, CInt]
 
+  // Some methods here have a @name annotation and some do not.
```

Comment:
I think a comment like this is better put in the docs until at some time we have better interoperability with structs and we don't need these gyrations. The problem is that we have plenty of other places where we have annotations including for macros. Docs with some simple examples is more work but could help when we get questions from all the new users. I even fell in this trap recently thinking I could do something without glue code. Also, now that third party libraries can add C glue code it makes it even more relevant.

Reply:
Eric

People of good will can differ in opinion.  To my thinking, the place where a 
developer will find the answer to a tangle of obvious inconsistency is in the code.

I am not suggesting War & Peace in the code. Such should probably be a 
one line reference to an Issue or commit SHA (or whatever git uses to replace SHA).

I know that I have spent several eternities, several times, trying to reverse-engineer
the practice or find any pattern to what gets an annotation and what does not.
I hate to admit it, but I think I have introduced a few bugs over the years when
I got it wrong.

The reality of these days is that information appears in the source file many months
before it appears in the Contributor's Guide in a readable form.  That is a lot
of time for well meaning people to introduce bugs.

Cheap enough to have the comment here.  To my thinking, it is a mote issue
when time.scala and posixlib have beam (or Alaska) size issues.

Reply:
I can remove the comment if that helps this PR move forward.